src/main/java/frc/robot/math/MobileShootCalculator.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.math;
+
+import java.util.ArrayList;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+
+public class MobileShootCalculator {
+  
+  private final ChassisSpeeds chassisSpeeds = new ChassisSpeeds();
+  ArrayList<Double> currentSpeeds = new ArrayList<Double>();
+
+  public MobileShootCalculator() {
```

Comment:
I don't think this class would work. The constructor doesn't set `currentSpeeds`. `currentSpeeds` is package-private (default visibility), so it seems like the intent might have been that a caller (in the same package) would directly set the field, but they wouldn't be able to do that before constructing the object. In short, you'd always end up with zero.